R/modules.R

```diff
@@ -103,6 +103,37 @@ append_module <- function(modules, module) {
   modules
 }
 
+#' Extract specific class from a list of `modules`
+#' @param modules `teal_modules`
```

Comment:
```suggestion
#' @param modules `teal_modules` or `teal_module`, whatever is passed to `init`'s `modules` argument
```
Will require adjustment when migrating to list of `teal_module`s.